# mavlink_mission_item_int

int16 seq
int16 total_seq  # original
int16 command
float64 latitude
float64 longitude
